lib/models/rlModel.js

```diff
@@ -0,0 +1,20 @@
+const pool = require('../utils/pool');
+module.exports = class RlGame {
+  constructor(row) {
+    this.id = row.id;
+    this.name = row.name;
+    this.name = row.platform;
```

Comment:
You have `this.name` twice which will cause the value of `this.name`  to overwritten by `row.platform` here.